styles.css

```diff
@@ -0,0 +1,131 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.header {
+  width: 375px;
+  height: 92px;
+  display: flex;
+  align-items: center;
+  background-color: #3c3a39;
+}
+
+.logo {
+  width: 77px;
+  height: 24px;
+  font-family: Roboto, sans-serif;
```

Comment:
- [ ] You must wrap `Roboto` in a quote before it could be detected. Like this 'Roboto'.